src/components/KaotoToolbar.tsx

```diff
@@ -47,18 +47,19 @@ import { useCallback, useEffect, useState } from 'react';
 
 export interface IKaotoToolbar {
   toggleCatalog: () => void;
+  leftDrawerExpanded:boolean;
```

Comment:
Tiny nitpick, but can you add a space here for better formatting? Like `leftDrawerExpanded: boolean`, please? Thanks!